about.html

```diff
@@ -12,62 +12,83 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+  <header>
+    <img class="logo" src="img/lambda-black.png">
+    <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+     </nav>
+    <img class="banner" src="img/jumbo-about.png">
+  </header>
+
+    <div id="container-about-page">
+
```

Comment:
The content area of the about.html page is a good place to use semantic HTML instead of using divs.  One example is using sections for this area.